docs/_static/niscope_usage.inc

```diff
@@ -21,7 +21,7 @@ The following is a basic example of using the **niscope** module to open a sessi
         # Find all record number 3
         rec3 = [wfm for wfm in waveforms if wfm.record == 3]
 
-The waveform returned from `fetch <http://nimi-python.readthedocs.io/en/master/niscope/functions.html#niscope.Session.fetch>`_ is a flat list of Python objects
+The waveform returned from `fetch <https://nimi-python.readthedocs.io/en/master/niscope/class.html#fetch>`_ is a flat list of Python objects
```

Comment:
Can we make these relative paths? so that the versioned docs point to their own documents?

Reply:
Changed to relative paths. Can't use :py:meth:`niscope.Session.fetch` link format (Sphinx) since this file is also used for PyPI